.changeset/smart-jeans-tickle.md

```diff
@@ -0,0 +1,24 @@
+---
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/address": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/example-contract": patch
+"forc-bin": patch
```

Comment:
Remove forc bin from the changesets. It will lead to a issue when the release is created.   Can you add on the changesets ignore also?

Reply:
It seems that by ignoring the `forc-bin` package triggers a cascade event in which we need to ignore several other packages that depend on it, namely:

 - `@fuel-ts/contract`
 - `@fuel-ts/example-contract"`
 - `@fuel-ts/predicate"`
 - `@fuel-ts/script"`
 - `forc-bin`
 - `fuel-gauge`
 - `fuels`
 - `typechain-target-fuels`

Reply:
@luizstacio Also, could you please expand on what issue it will lead to when creating the release?

Reply:
forc-bin package uses the version of the package to download the forc binary, in this way adding here on the changesets will try to download the wrong version.

Reply:
> It seems that ignoring the `forc-bin` package triggers a cascade event in which we need to ignore several other packages that depend on it, namely:
> 
> * `@fuel-ts/contract`
> * `@fuel-ts/example-contract`
> * `@fuel-ts/predicate`
> * `@fuel-ts/script`
> * `fuels`
> * `typechain-target-fuels`

Do you know why? And if is possible to fix this?